application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -494,17 +494,38 @@ define('diagram-store', ['jquery', 'xwiki-meta', 'xwiki-utils', 'diagram-utils',
         // container, meaning also for ul and ol elements.
         alt.removeAttribute('text-anchor');
 
+        // Adapt elements for XWiki PDF export since links are not displayed when 'a' tag is placed inside a 'text'
+        // element.
+        var isLinkTag = alt.tagName.toLowerCase() == 'a';
+
         // Have a bullet for each row.
-        var delimiter = document.createElementNS('http://www.w3.org/2000/svg', 'tspan');
+        var delimiter = document.createElementNS('http://www.w3.org/2000/svg', isLinkTag ? 'text' : 'tspan');
         delimiter.textContent = "•";
         delimiter.setAttribute('x', w - 2 * spaceWidth);
+        delimiter.setAttribute('y', h);
 
         // Move all to a container.
-        var container = document.createElementNS('http://www.w3.org/2000/svg', 'text');
+        var container = document.createElementNS('http://www.w3.org/2000/svg', isLinkTag ? 'g' : 'text');
         container.setAttribute('x', w);
         container.setAttribute('y', h);
         container.appendChild(delimiter);
+
+        if (isLinkTag) {
+          // The text inside alt is distributed to 'tspan' elements and since they are showned only if are placed
+          // under a 'text' parent, we add it between them and the link.
+          var text = document.createElementNS('http://www.w3.org/2000/svg', 'text');
+          text.appendChild($(alt).children()[0]);
```

Comment:
It seems you're moving only the first child. I would add a comment to explain why.